src/Setup/DbUtil.php

```diff
@@ -238,4 +240,29 @@ public static function findTables($conn, $databaseName) {
     }, self::fetchAll($conn, $sql));
   }
 
+  public function generateCreateSql($srcPath, $databaseName, $tables) {
```

Comment:
A general comment on `r-code` style -- these three functions (`generateCreateSql()`, `generateNavigation()`, and `generateSample()`) are qualitatively different from the other functions in this class (`parseDsn`, `encodeDsn`, `sourceSQL`). The older functions deal more the connection (making the connection; sending data to it), and the newer functions deal more with content (composing SQL documents).

IMHO, the newer functions merit their own class -- e.g. `SchemaGenerator::generateNavigation()`.